vmdk_plugin/drivers/vmdk/vmdk_driver.go

```diff
@@ -65,7 +60,7 @@ func NewVolumeDriver(port int, useMockEsx bool, mountDir string, driverName stri
 	if useMockEsx {
 		d = &VolumeDriver{
 			useMockEsx: true,
-			ops:        vmdkops.VmdkOps{Cmd: vmdkops.MockVmdkCmd{}},
+			ops:        vmdkops.VmdkOps{Cmd: vmdkops.NewMockCmd()},
```

Comment:
general request - please do not call thing "New".  In a very short time it will be old and will be confusing. Pick up semantically meaningful name please